libr/bin/p/bin_mach0.c

```diff
@@ -15,6 +16,15 @@ extern RBinWrite r_bin_write_mach0;
 
 static RBinInfo *info(RBinFile *bf);
 
+static void swizzle_io_read(struct MACH0_(obj_t) *obj, RIO *io);
+static int rebasing_and_stripping_io_read(RIO *io, RIODesc *fd, ut8 *buf, int count);
+static void rebase_buffer(struct MACH0_(obj_t) *obj, ut64 off, RIODesc *fd, ut8 *buf, int count);
+
+static RList * pending_bin_files = NULL;
```

Comment:
We dont want globals

Reply:
heh yeah this is a hack. not sure how to do otherwise without changing fundamental stuff. i can try to think something over the weekend.

Reply:
ok, managed to remove the global. had to do slight changes to bfile.c and bobjc.c, but according to r2r i didn't break anything.